mm2src/mm2_main/src/lp_swap/maker_swap.rs

```diff
@@ -576,6 +576,26 @@ impl MakerSwap {
             },
         };
 
+        // Validate maker_coin_htlc_pubkey realness
+        match self.taker_coin.validate_other_pubkey(taker_data.maker_coin_htlc_pub()) {
```

Comment:
We should call `validate_other_pubkey` for our `maker_coin` with the `taker_data.maker_coin_htlc_pub()` argument.